tests/cli/bootsys/test_cabinet_power.py

```diff
@@ -69,3 +78,95 @@ def test_do_cabinets_power_off_without_prompting(self):
 
         self.mock_liquid_cooled.assert_called_once_with(self.args)
         self.mock_air_cooled.assert_called_once_with(self.args)
+
+
+class TestRecreateCronjobs(unittest.TestCase):
+    def setUp(self):
+        now = datetime.datetime.now(tz=tzutc())
+        self.now = datetime.datetime(*now.timetuple()[:4], tzinfo=tzutc())
```

Comment:
A comment here that this is getting the current "top of the hour" time would be helpful